mlrun/datastore/model_provider/huggingface_provider.py

```diff
@@ -94,27 +96,45 @@ def _response_handler(
         **kwargs,
     ) -> Union[str, list, dict[str, Any]]:
         """
-        Same as `ModelProvider._response_handler`.
+        Processes and formats the raw response from the HuggingFace pipeline according to the specified format.
+
+        Expected to receive the response with `return_full_text=False`.
```

Comment:
What does this mean: "Expected"? That the response is typically with `return_full_text=False`.
I can help improve the sentence when I know what the intention is